src/cumulus/commands/start.py

```diff
@@ -1,8 +1,34 @@
-from .base import Base
+from .base import *
+import yaml
 
 
 class Start(Base):
 
     def run(self):
-        for service in self.options['<service>']:
-            Base.start_container(self.__class__.__name__, service)
+        if not self.options['<service>']:
+            start_container("")
+        else:
+            for service in self.options['<service>']:
+                start_container(service)
+        port = active_port()
+        if port == "":
+            print("Error: no port exposed in web app")
+        elif port == "none":
+            print("No web app to be shown")
+        else:
+            print("app available at localhost:"+port)
+
+
+def active_port():
```

Comment:
This function won't work if we ever use randomly-chosen host ports... Do you think we should make a note of that somewhere?

Reply:
Fixed for dynamic ports